modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala

```diff
@@ -257,6 +258,15 @@ object EpicsUtil {
   def safeAttributeSInt[F[_]: Sync, A](get: => CaAttribute[JInt]): F[Option[Int]] =
     Nested(safeAttribute(get)).map(_.toInt).value
 
+  def safeAttributeList[F[_]: Sync, A](get: => CaAttribute[A]): F[Option[util.List[A]]] =
```

Comment:
why `util.List`?

Reply:
I wonder if you could represent this as just a `List` with the `None` being an empty List.
Otherwise perhaps it should be `Option[NonEmptyList[A]]`
Maybe you need to differentiate if the list was not set or empty?

Reply:
Yes, I prefer to be able to differentiate between those cases.